src/ir/namespace.cpp

```diff
@@ -58,26 +59,23 @@ NamedType* Namespace::newNamedType(string name, string nameFlip, Type* raw) {
   return named;
 }
 
-//TODO
-void Namespace::newNominalTypeGen(string name, string nameFlip, Params genparams, TypeGenFun fun) {
-  //Make sure the name and its flip are different
-  assert(name != nameFlip);
-  //Verify this name and the flipped name do not exist yet
-  assert(!typeGenList.count(name) && !typeGenList.count(nameFlip));
-  assert(!namedTypeList.count(name) && !namedTypeList.count(nameFlip) );
- 
-  //Add name to typeGenNameMap
-  typeGenNameMap[name] = nameFlip;
-  typeGenNameMap[nameFlip] = name;
-
-  //Create the TypeGens
-  TypeGen* typegen = new TypeGenFromFun(this,name,genparams,fun,false);
-  TypeGen* typegenFlip = new TypeGenFromFun(this,nameFlip,genparams,fun,true);
-  
-  //Add typegens into list
-  typeGenList[name] = typegen;
-  typeGenList[nameFlip] = typegenFlip;
-}
+//TODO remove this code
+//void Namespace::newNominalTypeGen(string name, string nameFlip, Params genparams, TypeGenFun fun) {
+//  //Make sure the name and its flip are different
+//  assert(name != nameFlip);
+//  //Verify this name and the flipped name do not exist yet
+//  assert(!typeGenList.count(name) && !typeGenList.count(nameFlip));
+//  assert(!namedTypeList.count(name) && !namedTypeList.count(nameFlip) );
+// 
+//  //Add name to typeGenNameMap
+//  typeGenNameMap[name] = nameFlip;
+//  typeGenNameMap[nameFlip] = name;
+//
+//  //Create the TypeGens
+//  TypeGenFromFun::make(this,name,genparams,fun,false);
+//  TypeGenFromFun::make(this,nameFlip,genparams,fun,true);
+//  
+//}
```

Comment:
Just remove it now? You can always go back to a commit to find it (if you give your commit an easily identifiable message)